app/javascript/src/NewService/components/NewServiceForm.jsx

```diff
@@ -3,19 +3,23 @@
 import React, {useState} from 'react'
 
 import {ServiceSourceForm, ServiceDiscoveryForm, ServiceManualForm} from 'NewService'
+import {FormWrapper, BackendApiSelect} from 'NewService/components/FormElements'
+import type {BackendApi} from 'NewService/types'
```

Comment:
This is really just a personal preference but I think it's better to import types after all the JS imports, since they are not really a real Javascript thing.